minitwit_backend/scripts/entrypoint.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/sh
```

Comment:
either they should both run shell, or both run bash, currently they one runs shell, the other one bash